docker/grafana.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM grafana/grafana
+USER root
+COPY docker/grafana/setup-grafana /usr/share/grafana/
+COPY docker/grafana/setup-grafana.sh /usr/local/bin/
+COPY docker/grafana/grafana.ini /etc/grafana/
+COPY docker/grafana/provisioning/datasources/* /etc/grafana/provisioning/datasources/
+COPY docker/grafana/provisioning/dashboards/* /etc/grafana/provisioning/dashboards/
```

Comment:
Is there a way to install dashboards to where rpm installs them and then not have to have docker specific ones?

Reply:
I don't believe I can use rpm here but I may be able to use the existing dashboard config.